_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,33 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  --------------------------------
+## MD029 mod - tells linter not to fail if order list items don't start with 1
```

Comment:
Maybe the comment isn't very clear here, was trying to think of a way to succinctly say it...

Basically, it tells the linter that it is okay to have ordered lists that increment, which seems like it should be the default, but the default for the linter is it is expecting each ordered list item to start with 1.  For example:

By default the linter expects ordered list items to be like below:

    1. Do this.
    1. Do that.
    1. Done.

However, we have list items like below that this configuration then allows::

    1. Do this.
    2. Do that.
    3. Done.